front/components/assistant_builder/InstructionScreen.tsx

```diff
@@ -98,6 +99,31 @@ export function InstructionScreen({
   ) => void;
   setEdited: (edited: boolean) => void;
 }) {
+  const editor = useEditor({
+    extensions: [
+      StarterKit.configure({
+        heading: false,
+      }),
+    ],
+    content: builderState.instructions?.replaceAll("\n", "<br />") || "",
```

Comment:
Why do we need this? For backward compatibility?

Reply:
No, just because we don't store instructions as HTML

Reply:
By using `editor.getText()` you'll get some plain text an lose all styles (probably also including carriage returns and so on), no?

Reply:
No get text gives you the newlines. We don't want any of the styling for now. And when you set the content initially it has to be HTML